lib/sampler/storage/src/test/span-dao-test.js

```diff
@@ -359,19 +362,23 @@ describe('span-dao', () => {
       await thirdSpanPO.updateSpanProcessedInterval(thirdInterval, true, 1).verify();
     });
 
-    it('returns only spans that are sampled earlier than specified', async () => {
+    it('returns only spans that are sampled earlier than specified and end is not set', async () => {
```

Comment:
Might be a good idea to have the test inside context('when end is not set') to have simple sentences